code/game/objects/items/explosives/mine.dm

```diff
@@ -220,56 +242,94 @@ taking that kind of thing into account, setting buffer_range = 0 or making them
 
 ///The process for trying to disarm a mine
 /obj/item/mine/proc/bomb_defusal(mob/user)
-	if(triggered && !interruptible)
+	if(triggered && !CHECK_BITFIELD(mine_features, MINE_INTERRUPTIBLE))
 		balloon_alert(user, "Can't disarm this, run!")
 		return FALSE
+
 	//Mine defusal time is de/increased by 1 second per skill point
 	var/skill_issue = max(disarm_delay - user.skills.getRating(SKILL_ENGINEER) + SKILL_ENGINEER_ENGI, 0)
+
+	//Chance for failure below; only /living/ types can be checked for brain damage
+	var/mob/living/living = isliving(user) ? user : null
+	var/homer_simpson = living ? living.brainloss : 0
```

Comment:
pls

Reply:
I did it this way so that if you got a little bit of brain damage, while you might be super skilled, you still have a chance to fail because you made a mistake you did not mean to. If you are unskilled AND have brain damage, _please_ do not go around defusing mines.

Reply:
Name this var properly